### PR TITLE
Add Span.setSpanName().

### PR DESCRIPTION
**Pull request by carlosalberto**
Required to support `io.opentracing.Span.setOperationName()` in the OT bridge.

### CODE REVIEW

File: api/src/main/java/openconsensus/trace/BlankSpan.java [hunk 1]
```diff
@@ -89,6 +89,11 @@ public void addLink(Link link) {
     Utils.checkNotNull(link, "link");
   }
 
+  @Override
+  public void setSpanName(String spanName) {
```
Comment (SergeyKanzhelev) [inline api/src/main/java/openconsensus/trace/BlankSpan.java]: yep. It is aligned with the changes we made in C# and issue: https://github.com/census-instrumentation/opencensus-specs/issues/217

**Comment by carlosalberto** [comment]
Addresses #23

File: api/src/main/java/openconsensus/internal/Utils.java [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package openconsensus.internal;
+
+/** General internal utility methods. */
+public final class Utils {
```
Comment (yurishkuro) [inline api/src/main/java/openconsensus/internal/Utils.java]: nit: could move many methods to a class called `Check`, e.g. `Check.argument()`, `Check.notNull()`.

**Comment by carlosalberto** [comment]
Closed on behalf of #27

**Pull request closed without merging**